scaleway-python/__main__.py

```diff
@@ -0,0 +1,27 @@
+import pulumi
+import pulumiverse_scaleway as scaleway
```

Comment:
Import is wrong, should be `lbrlabs_scaleway`

https://pypi.org/project/lbrlabs-scaleway/

Reply:
aaaaaah, sorry! Done!